integrationtests/tests/tests.rs

```diff
@@ -511,7 +551,9 @@ async fn lightning_gateway_pays_internal_invoice() -> Result<()> {
         gateway.user.assert_total_notes(sats(2010)).await; // gateway routed internally and earned fee
         receiving_user.assert_total_notes(sats(1000)).await; // this user received the 1000 sat invoice
 
-        assert_eq!(lightning.amount_sent().await, sats(0)); // We did not route any payments over the lightning network
+        if !lightning.is_shared() {
```

Comment:
This is necessary because it is possible another test sent some sats at the same time?

Reply:
I think so, yes.